webapp/src/pages/Settings.tsx

```diff
@@ -362,6 +395,64 @@ const Settings: React.FC<Props> = ({ open, onClose }) => {
       ),
     });
 
+  const displayPipelineAddSection = (pipelineIndex: number) => (
+    <Divider sx={{ position: "relative", height: 2 }}>
+      <Tooltip title="Add pipeline">
+        <IconButton
+          sx={{
+            bottom: 25,
+            marginTop: 2,
+            padding: 0,
+          }}
+          aria-label="add-pipeline"
+          disabled={isUpdatingConfig}
+          onClick={() =>
+            updatePartialConfig({
+              pipelines: [
+                ...resultingConfig.pipelines!.slice(0, pipelineIndex + 1),
+                ...defaultConfig.pipelines!,
+                ...resultingConfig.pipelines!.slice(pipelineIndex + 1),
+              ],
+            })
+          }
+        >
+          <AddCircleOutline />
+        </IconButton>
+      </Tooltip>
+    </Divider>
+  );
+
+  const displayPostprocessorAddSection = (
+    pipelineIndex: number,
+    postprocessorIndex: number,
+    postprocessor: PipelineDefinition["postprocessors"]
+  ) => (
+    <Divider textAlign="left" sx={{ position: "relative", height: 2 }}>
+      <Tooltip title="Add postprocessor">
+        <IconButton
+          sx={{
+            bottom: 25,
+            marginTop: 2,
+            padding: 0,
+          }}
```

Comment:
I notice that if you remove the `position` and the `height` on the `Divider`, as well as the `bottom`, the `marginTop`, and the `padding` on the `IconButton`, the button centers properly. I think it looks good with the `size="small"`.
```suggestion
    <Divider textAlign="left">
      <Tooltip title="Add postprocessor">
        <IconButton
          size="small"
```

Reply:
I prefer reducing the gap between the divider and the button by styling the divider component, with `[`& .${dividerClasses.wrapper}`]: { padding: 0 }`. We can also better align the button to the left (instead of the `textAlign="left"` that looked weird):
```suggestion
    <Divider
      sx={{
        "&::before": { width: (theme) => theme.spacing(2) },
        [`& .${dividerClasses.wrapper}`]: { padding: 0 },
      }}
    >
      <Tooltip title="Add postprocessor">
        <IconButton
```

Reply:
Note that I said
> instead of the `textAlign="left"` that looked weird

You can remove the `textAlign="left"` now.